docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: Grasa-Django-App
+    network_mode: host
+    ports:
+      - 80:80
+      - 443:443
+  db:
+    image: mysql:latest
+    command: mysql --default-authentication-plugin=mysql_native_password
+    volumes:
+      - "./mysql:/var/lib/mysql"
+    ports:
+      - 3306:3306
+    environment:
+      - MYSQL_ROOT_PASSWORD=djangoGrasa2019
+      - MYSQL_DATABASE=grasa_event_locator
+      - MYSQL_USER=grasaadmin
+      - MYSQL_PASSWORD=djangoGrasa2019
```

Comment:
Nice, this is the best place to set these :+1: They can also be overriden on the command line.